mmagic/models/editors/stable_diffusion/stable_diffusion.py

```diff
@@ -140,6 +143,24 @@ def set_tomesd(self) -> nn.Module:
     def device(self):
         return next(self.parameters()).device
 
+    def train(self, mode: bool = True):
```

Comment:
`DreamBooth` is inherited  from `StableDiffusion`, will this change influence the training / inference of `DreamBooth`?

Reply:
No, it looks correct.
![image](https://github.com/open-mmlab/mmagic/assets/49083766/85e4a699-5784-4109-9f18-320f2dd5aabc)
![image](https://github.com/open-mmlab/mmagic/assets/49083766/1f25808d-8443-4297-b559-6f9fde441e09)